contracts/RelayHubApi.sol

```diff
@@ -21,19 +21,23 @@ contract  RelayHubApi {
      * the client uses this event so it can report correctly transaction complete (or revert) to the application.
      * Monitoring tools can use this event to detect liveliness of clients and relays.
      */
-    event TransactionRelayed(address indexed relay, address indexed from, address indexed to, bytes32 hash, uint status, uint charge);
+    event TransactionRelayed(address indexed relay, address indexed from, address indexed to, uint status, uint charge);
     event Deposited(address src, uint amount);
     event Withdrawn(address dest, uint amount);
     event Penalized(address indexed relay, address sender, uint amount);
 
-    function get_nonce(address from) view external returns (uint);
-    function relay(address from, address to, bytes memory encoded_function, uint transaction_fee, uint gas_price, uint gas_limit, uint nonce, bytes memory approval) public;
+    function getNonce(address from) view external returns (uint);
+
+    function relay(address from, address to, bytes memory encodedFunction, uint transactionFee, uint gasPrice, uint gasLimit, uint nonce, bytes memory approval) public;
```

Comment:
should be renamed `relayCall`
- its better to have experssive method names, with object-subject
- we use the terminology of acceptRelayedCall, postRelayedCall, so it makes sense to use Call here too.

Reply:
done